### PR TITLE
Reorganize settings adv and fix spacing

### LINKED ISSUE #447: Put configs above disks in advanced page

### LINKED ISSUE #451: Fix spacing on advanced page

### PR DESCRIPTION
**Pull request by na3d**
Closes #451
Closes #447

Did these at the same time since one would affect the other.

### CODE REVIEW

**Comment by na3d** [comment]
screen shot 2016-09-14 at 9 58 24 am

File: src/linodes/settings/layouts/AdvancedPage.js [hunk 1]
```diff
@@ -6,13 +6,13 @@ import { DiskPanel } from '~/linodes/settings/components/DiskPanel';
 export function AdvancedPage(props) {
   return (
     <div className="form-group">
-      <div className="content-col">
+      <div className="content-col form-group">
```
Comment (eatonphil) [inline src/linodes/settings/layouts/AdvancedPage.js]: There is already a form-group right above this. This should not be here.

Comment (na3d) [inline src/linodes/settings/layouts/AdvancedPage.js]: Nice catch!

Comment (na3d) [inline src/linodes/settings/layouts/AdvancedPage.js]: By removing it in the parent div, it keeps the look the same.

Comment (eatonphil) [inline src/linodes/settings/layouts/AdvancedPage.js]: I'm not sure what you mean. If you need this form-group here for the sake of it's margin-bottom, in addition to the form-group parent in line 8, then you should still not use form-group again here and style it differently. This is not what form-group is for.

**Pull request merged**